run.py

```diff
@@ -6,4 +7,4 @@
 app = create_app(config)
 
 if __name__ == '__main__':
-    app.run(debug=True, port=5006)
+    app.run(debug=True, port=5000)
```

Comment:
I like the choice to move to a commonly used port.

Reply:
OK let me put the port to be 5000